tests/baselines/reference/parseNoWhitespaceBeforeExclamation.errors.txt

```diff
@@ -0,0 +1,42 @@
+parseNoWhitespaceBeforeExclamation.ts(3,6): error TS1289: A non-null assertion may not be preceded by whitespace.
+parseNoWhitespaceBeforeExclamation.ts(4,6): error TS1289: A non-null assertion may not be preceded by whitespace.
+parseNoWhitespaceBeforeExclamation.ts(5,6): error TS1289: A non-null assertion may not be preceded by whitespace.
+parseNoWhitespaceBeforeExclamation.ts(6,6): error TS1289: A non-null assertion may not be preceded by whitespace.
+parseNoWhitespaceBeforeExclamation.ts(7,6): error TS1289: A non-null assertion may not be preceded by whitespace.
+parseNoWhitespaceBeforeExclamation.ts(8,6): error TS1289: A non-null assertion may not be preceded by whitespace.
+parseNoWhitespaceBeforeExclamation.ts(9,6): error TS1289: A non-null assertion may not be preceded by whitespace.
+parseNoWhitespaceBeforeExclamation.ts(10,6): error TS1289: A non-null assertion may not be preceded by whitespace.
+parseNoWhitespaceBeforeExclamation.ts(11,6): error TS1289: A non-null assertion may not be preceded by whitespace.
+
+
+==== parseNoWhitespaceBeforeExclamation.ts (9 errors) ====
+    declare var value: any;
+    
+    value !;
+         ~~
+!!! error TS1289: A non-null assertion may not be preceded by whitespace.
+    value !instanceof String;
+         ~~
+!!! error TS1289: A non-null assertion may not be preceded by whitespace.
+    value ! instanceof String
+         ~~
+!!! error TS1289: A non-null assertion may not be preceded by whitespace.
+    value/* this is a comment */!instanceof String;
+         ~~~~~~~~~~~~~~~~~~~~~~~~
+!!! error TS1289: A non-null assertion may not be preceded by whitespace.
+    value/* this is a comment */! instanceof String;
+         ~~~~~~~~~~~~~~~~~~~~~~~~
+!!! error TS1289: A non-null assertion may not be preceded by whitespace.
+    value !in String;
+         ~~
+!!! error TS1289: A non-null assertion may not be preceded by whitespace.
+    value ! in String
+         ~~
+!!! error TS1289: A non-null assertion may not be preceded by whitespace.
+    value/* this is a comment */!in String;
+         ~~~~~~~~~~~~~~~~~~~~~~~~
+!!! error TS1289: A non-null assertion may not be preceded by whitespace.
+    value/* this is a comment */! in String;
+         ~~~~~~~~~~~~~~~~~~~~~~~~
+!!! error TS1289: A non-null assertion may not be preceded by whitespace.

```

Comment:
Obviously this is not the way I want to emit this error; we can discuss this later of course.

Reply:
What's wrong with it? The inclusion of the `!` itself?

Reply:
Complaining about a comment when there's no whitespace; I would have expected at least one tool to support some sort of interleaved comment a la `/*@__PURE__*/` or something. Maybe that's silly. If we do want to complain about comments, then my message would need to be updated (which is fine).